### ISSUE TITLE
Gencode database uri in fusioncatcher-build is outdated after moving to ebi.co.uk

### ISSUE THREAD

**Issue by ilykos** [ndaniel/fusioncatcher#114]
> --------------------------------------------------------------------------------
> +-->EXECUTING...
> Downloading the GTF file of organism 'mus_musculus' from Gencode!
> 230 Anonymous access granted, restrictions apply
> FTP Error = 550 pub/gencode/Gencode_mouse/: No such file or directory
> 
> 
> ERROR: Workflow execution failed at step 22 while executing:
> 
>    get_gencode.py \
>    --organism mus_musculus \
>    --output /home/yin/fusioncatcher/data/mouse/
> 
> 
> 
> Executing second time the same step/command in order to capture error messages (i.e. STDERR)...
> 
> -------------------------------------------
> Downloading the GTF file of organism 'mus_musculus' from Gencode!
> 230 Anonymous access granted, restrictions apply
> FTP Error = 550 pub/gencode/Gencode_mouse/: No such file or directory

The database link in **get_gencode.py** is outdated.

@ndaniel Since gencode was moved to ftp.ebi.ac.uk. Please introduce following changes in **get_gencode.py**:

```
    parser.add_option("--server",
                      action="store",
                      type="string",
                      dest="server",
                      default = "ftp.ebi.ac.uk", '%default'.""") # <--- correct default address
```

```
   if org:

        url = 'pub/databases/gencode/%s/' % (org,)0 <--- correct folder structure on the new gencode ftp
```

This will fix the **fusioncatcher-build.py** script.

**Comment by ndaniel** [comment]
Thanks!
This will be fixed in release 1.10 which will be released in few days!

**Comment by ndaniel** [comment]
Fixed in release 1.10.
